src/averageAges.js

```diff
@@ -20,8 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
-}
+  let men = people.filter(obj => {
+    return obj['sex'] === 'm';
+  });
 
+  if (century) {
+    men = men.filter(obj => Math.ceil(obj.died / 100) === century);
+  };
```

Comment:
use ternary operator instead of resign value. Apply for other cases as well
Ex: 
```
const yourVariable = array.filter(val => 
    century
      ? {{some_code}}
      : {{some_code}}
)
```